README.md

```diff
@@ -70,6 +70,13 @@ The directives have the following options:
     Button()
 ```
 
+### Configuration
+
+You conf.py has two extra configuration options:
+
+ * jupyter_sphinx_require_url: url for `require.js` (if your theme already provides this, set it to False or '')
```

Comment:
Ah, the docs there seemed to indicate that requirejs must be on the page. Let's reword, then. Perhaps:

```
* jupyter_sphinx_require_url: url for `require.js` (if requirejs is not needed or your theme already provides this, set it to False or '')
```
How do I say that my theme already provides require, so I must use the require embedding URL?

How about the following options:

1. jupyter_sphinx_use_requirejs = True / False, defaults to True
2. jupyter_sphinx_requirejs_url = some url, or the default of '' to automatically include
3. jupyter_sphinx_embed_url = some url, or the default of '' to automatically choose an embed url based on jupyter_sphinx_use_requirejs